File: app/src/main/java/com/nytsample/android/data/model/NewsFeedResponseModel.java
package com.nytsample.android.data.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class NewsFeedResponseModel implements Serializable {
    public String status = "";

    @SerializedName("num_results")
    public int numOfResults = 0;

    public List<NewsFeedItemModel> results = new ArrayList<>();
}

File: app/src/main/java/com/nytsample/android/data/model/NewsFeedResponseModel.kt
package com.nytsample.android.data.model

import com.google.gson.annotations.SerializedName

class NewsFeedResponseModel {
    var status: String = ""

    @SerializedName("num_results")
    var numOfResults: Int = 0

    var results: List<NewsFeedItemModel> = emptyList()
}